src/driver/libxdma.c

```diff
@@ -1407,6 +1402,16 @@ static int irq_setup(struct xdma_dev *xdev)
 	return 0;
 }
 
+static void irq_teardown(struct xdma_dev *xdev)
+{
+	irq_msix_channel_teardown(xdev);
+}
+
+static int irq_setup(struct xdma_dev *xdev)
+{
+	return xdma_irq_setup(xdev);
+}
```

Comment:
I don't understand why we need these tiny wrappers that just confuse the names? Can't you just rename the call sites?

Reply:
Future patches move irq_setup() out of libxdma.c and add the eBPF irq setup to it.

However, a function on libxdma.c (xdma_device_online(), which is called by slot_reset()) still needs to call irq_setup(), but without the future eBPF stuff.